cmd/vic-machine/create/create.go

```diff
@@ -864,17 +875,47 @@ func (c *Create) processDNSServers() error {
 	return nil
 }
 
-func (c *Create) processVolumeStores() error {
+func (c *Create) processDefaultVolumeStore() error {
 	defer trace.End(trace.Begin(""))
+	//we make this map here since it is used many times after.
 	c.VolumeLocations = make(map[string]string)
+	if c.DefaultVolumeLocation != "" {
+		c.VolumeLocations[defaultVolumeLabel] = c.DefaultVolumeLocation
+	}
+	return nil
+}
+
+func (c *Create) processVolumeStores() error {
+	defer trace.End(trace.Begin(""))
 	for _, arg := range c.volumeStores {
+
 		splitMeta := strings.SplitN(arg, ":", 2)
 		if len(splitMeta) != 2 {
 			return errors.New("Volume store input must be in format datastore/path:label")
```

Comment:
volume store format could be datastore:lable as well. Might update this error message, please?